site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -875,6 +903,124 @@ This strategy instructions FusionAuth when to append a unique suffix to the user
++
+ * `any` - An authenticator with any attachment modality will be allowed during registration.
+ * `crossPlatform` - Only authenticators with the `cross-platform` attachment modality will be allowed during registration. These are also referred to as "roaming" authenticators.
+ * `platform` - Only authenticators with the `platform` attachment modality will be allowed during registration.
++
+The recommended value for the reauthentication workflow is `platform`.
++
+:advanced_feature: WebAuthn
+include::../shared/_premium-edition-blurb-api.adoc[]
++
+:advanced_feature!:
++
+:enterprise_feature: WebAuthn roaming authenticators
+include::../shared/_enterprise-edition-blurb-api.adoc[]
++
+:enterprise_feature!:
+
+[field]#tenant.webAuthnConfiguration.reauthenticationWorkflow.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.41.0#::
+Whether or not this tenant has the WebAuthn reauthentication workflow enabled. The reauthentication workflow will automatically prompt a user to authenticate using WebAuthn on repeat logins from the same device.
++
+:advanced_feature: WebAuthn
+include::../shared/_advanced-edition-blurb-api.adoc[]
++
+:advanced_feature!:
+
+[field]#tenant.webAuthnConfiguration.reauthenticationWorkflow.userVerificationRequirement# [type]#[String]# [optional]#Optional# [default]#defaults to `required`# [since]#Available since 1.41.0#::
+Determines the user verification requirement for WebAuthn passkey registration and authentication when using the reauthentication workflow. The possible values are:
++
+ * `discouraged` - If possible, do not require user verification during registration and authentication.
+ * `preferred` - Prefer authenticators that support user verification during registration and authentication but allow others.
+ * `required` - Only authenticators that support user verification will be allowed.
++
+It is _highly_ recommended to use the `required` option for the reauthentication workflow.
++
+:advanced_feature: WebAuthn
+include::../shared/_advanced-edition-blurb-api.adoc[]
++
+:advanced_feature!:
+
+[field]#tenant.webAuthnConfiguration.relyingPartyId# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**# [since]#Available since 1.41.0#::
+The value this tenant will use for the Relying Party Id in WebAuthn ceremonies. Passkeys can only be used to authenticate on sites with the same Relying Party Id they were registered. This value must match your fully qualified domain name or be a registrable suffix of your fully qualified domain. For example, if your domain is `auth.example.com`, you could use `auth.example.com` or `example.com` but not `m.auth.example.com` or `com`.
++
+When this parameter is omitted, FusionAuth will use `null` for the Relying Party Id in passkey creation and request options. A `null` value in the WebAuthn client API will use the current fully qualified domain.
```

Comment:
The "current fully qualified domain" of what? The FusionAuth instance, I presume.

Reply:
I wasn't sure how to phrase this. It's the FQDN of the _browser_ when the user authenticates.

So if they are calling FusionAuth APIs at `company-prod.fusionauth.io`, but the login page is at `auth.company.com`, it would be `auth.company.com`.

Any suggestions for a succinct way to say that?

Reply:
The spec uses the following to describe the behavior when the parameter is omitted from the call to the WebAuthn client API:
> By default, the RP ID for a WebAuthn operation is set to the caller’s origin's effective domain.

Reply:
Changed this to use "browser origin," which is closer to the spec and more correct.